mtags/src/main/scala/scala/meta/internal/docstrings/MarkdownGenerator.scala

```diff
@@ -104,9 +107,14 @@ object MarkdownGenerator {
       else "",
       if (c.see.nonEmpty)
         "\n**See**\n" + c.see
-          .map(
-            body => "- [" ++ blocksToMarkdown(body.blocks).trim + "]()"
-          )
+          .map { body =>
+            // If we already have a link, just render it
+            // otherwise create a link from the given text
+            blocksToMarkdown(body.blocks).trim match {
+              case LinkPattern(text) => s"- $text"
+              case text => s"- [$text]($text)"
```

Comment:
```suggestion
              case text => s"- [`$text`]($text)"
```

How about this pattern? It should look like a verbatim text which is also a link.